packages/sanity-suite/__fixtures__/sourceConfig1.json

```diff
@@ -1,4 +1,16 @@
 {
+  "consentManagementMetadata": {
```

Comment:
this will break the tests untill the updated sourceConfig response is deployed right?

Reply:
Yes, but we can merge and deploy only after the config backend service is deployed.

Reply:
perfect